BUILD.bazel

```diff
@@ -0,0 +1,143 @@
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+load("@rules_oci//oci:defs.bzl", "oci_image", "oci_push", "oci_tarball")
+
+#
+# Counter-Strike: Source Layer
+#
+
+pkg_tar(
+    name = "counterstrikesource_layer",
+    srcs = ["@counter_strike_source//:files"],
+    package_dir = "/opt/game",
+)
+
+#
+# Maps Layer
+#
+
+pkg_tar(
+    name = "maps_layer",
+    package_dir = "/opt/game/cstrike",
+    deps = [
+        "@maps//file",
+        "@maps_bz2//file",
+    ],
+)
+
+#
+# MetaMod Layer
+#
+
+pkg_tar(
+    name = "metamod_layer",
+    deps = ["@metamod//file"],
+    package_dir = "/opt/game/cstrike",
+)
+
+#
+# SourceMod Layer
+#
+
+pkg_tar(
+    name = "sourcemod_layer",
+    deps = ["@sourcemod//file"],
+    empty_dirs = ["/opt/game/cstrike/addons/souremod/configs"],
+    package_dir = "/opt/game/cstrike",
+)
+
+#
+# Authorization Layer
+#
+
+pkg_tar(
+    name = "authorization_layer",
+    package_dir = "/opt/game/cstrike",
+    deps = ["@auth_by_steam_group//file"],
+)
+
+#
+# Plugins Layers
+#
+
+pkg_tar(
+    name = "plugins_layer",
+    package_dir = "/opt/game/cstrike",
+    deps = [
+        "@disable_buyzones//file",
+        "@disable_radar//file",
+        "@disable_round_timer//file",
+        "@ffa_spawns//file",
+        "@free_for_all//file",
+        "@gungame//file",
+        "@map_settings//file",
+        "@max_cash//file",
+        "@paintball//file",
+        "@remove_objectives//file",
+        "@respawn//file",
+        "@spawn_protection//file",
+    ],
+)
+
+#
+# Config Layer
+#
+
+pkg_tar(
+    name = "config_layer",
+    srcs = [":server.cfg"],
+    package_dir = "/opt/game/cstrike/cfg/templates",
+)
+
+# TODO: libcurl
```

Comment:
I'd prefer not to merge this until we have a solution for this so that we don't need to rely on having a server password.